tests/tests.coffee

```diff
@@ -93,6 +93,49 @@ class VexTabTests
     tab.getArtist().render(renderer)
     assert.ok(true, "all pass")
 
+  makeCanvas = (vex, test_id, flex)->
```

Comment:
Can you move `makeCanvas` and `renderCodeInCanvasId` into the function body of `getRenderedContent` (since they're only used there.) You can keep them as functions with the same names.

Reply:
Updated, thank you.  Also makes these functions smaller, I'm using the vars from the closure.